azrcrv-redirect.php

```diff
@@ -299,6 +300,7 @@ function display_options() {
 		</h1>
 		<?php
 
+		// phpcs:ignore WordPress.Security.NonceVerification.Recommended
```

Comment:
Adding an explanatory comment above this line might be helpful:

```suggestion
		// There is a 'manage-options' capability check at the top of this function.
		// phpcs:ignore WordPress.Security.NonceVerification.Recommended
```